pkg/flowaggregator/exporter/ipfix_test.go

```diff
@@ -288,3 +291,97 @@ func TestIPFIXExporter_sendRecord_Error(t *testing.T) {
 
 	assert.Error(t, ipfixExporter.AddRecord(mockRecord, false))
 }
+
+func createElementList(isIPv6 bool, mockIPFIXRegistry *ipfixtesting.MockIPFIXRegistry) {
```

Comment:
looks like there is code duplication with the existing `TestIPFIXExporter_sendTemplateSet` test case. When `TestIPFIXExporter_sendTemplateSet` could be updated to use this new helper function.